content/en/digital_certificate.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Digital Certificate
+status: Feedback Appreciated
+category: Concept
+---
+
+## What it is
+Digital certificates or public key certificates are issued by a certificate authority (CA). These certificates are electronic files or passwords that allow entities to exchange data or resources over the internet securely.
+
+Certificates authenticate the user through encryption methods and a set of roles and policies. Just as a passport certifies an individual's identity, a digital certificate establishes the user's identity. Note that a user can be a person or service within a [microservice](https://glossary.cncf.io/microservices/) app. 
```

Comment:
I would like to drop line 10. This goes into a specific use case and isn't sufficiently general.